packages/amplication-build-manager/src/build-logger/build-logger.controller.ts

```diff
@@ -3,7 +3,7 @@ import { Body, Controller, Post } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { Env } from "../env";
```

Comment:
ConfigService and env are not used anymore.

Let's also remove the dependency from the constructor